### ISSUE TITLE
Update included Bundle Analyzer with new treemap support

### ISSUE THREAD

**Issue by janpio** [fabiospampinato/banal#4]
https://esbuild.github.io/analyze/ recently got an update to include a nice treemap:
https://github.com/esbuild/esbuild.github.io/commit/2ead9e4b5b33347cf1a288cf757e0fe8052b009e
https://github.com/evanw/esbuild/commit/65b305894dc8bb3bae8a2a2c88f2b461135ef13f

Would be neat if this was also available here.

**Comment by janpio** [comment]
I am not quite sure how you got from https://github.com/esbuild/esbuild.github.io/tree/main/src/analyze or https://esbuild.github.io/analyze/ to the included file here, otherwise I would also open an PR for that change.

**Comment by fabiospampinato** [comment]
I'd love to get this and the other issues sorted, I'm pretty busy with other projects at the moment, I'm gonna need a ~week before I can work on these.

**Comment by janpio** [comment]
Of course. This one here is the only one where I just do not know what needs to be done. Can you maybe describe how you produced the current https://github.com/fabiospampinato/banal/blob/master/resources/analyzer.html - then I am more than happy to do that myself and open a PR and safe you the work.

**Comment by fabiospampinato** [comment]
If you save that page you'll see that it can load the metadata file from the a query parameter or something, near the end of the loaded JS, I'm just inserting the base64-encoded metadata file in there.

**Comment by fabiospampinato** [comment]
I've just released v1.4, which:

- Supports rendering a treemap.
- Can analyze a metafile directly with a `--metafile foo.json`. (reimplemented rather merging your PR, sorry it was just a bit quicker)
- Can output all the generated files to a custom path with a `--output out`.
- Can output just the metafile somewhere with a `--output-metafile metafile.json`.
- Can output just the analysis somewhere with a `--output-analysis analysis.html`.

Basically it should address everything you opened issues for 👍

**Comment by janpio** [comment]
Amazing, thanks so much! Will try it out soon.
